.dotnet/src/Custom/Chat/ChatAudioOptions.cs

```diff
@@ -0,0 +1,28 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+namespace OpenAI.Chat;
+
+/// <summary>
+/// Represents the configuration details for output audio requested in a chat completion request.
+/// </summary>
+/// <remarks>
+/// When provided to a <see cref="ChatCompletionOptions"/> instance's <see cref="ChatCompletionOptions.AudioOptions"/> property,
+/// the request's specified content modalities will be automatically updated to reflect desired audio output.
+/// </remarks>
+[CodeGenModel("CreateChatCompletionRequestAudio")]
+public partial class ChatAudioOptions
+{
+    /// <summary>
+    /// Gets or sets the voice model that the response should use to synthesize audio.
+    /// </summary>
+    [CodeGenMember("Voice")]
+    public ChatOutputAudioVoice OutputAudioVoice { get; set; }
+
+    /// <summary>
+    /// Specifies the output format desired for synthesized audio.
+    /// </summary>
+    [CodeGenMember("Format")]
+    public ChatOutputAudioFormat OutputAudioFormat { get; set; }
+}
```

Comment:
nit: Could you add "CUSTOM" comments to both properties. If I'm reading this right, the only customization is that they were renamed?